namespace.go

```diff
@@ -246,13 +261,16 @@ func (ns *Namespace) getUserAgent() string {
 	return userAgent
 }
 
+func (ns *Namespace) getResourceURI() string {
+	if ns.Environment.ResourceIdentifiers.ServiceBus == "" {
+		return serviceBusResourceURI
+	}
+	return ns.Environment.ResourceIdentifiers.ServiceBus
+}
+
 func (ns *Namespace) resolveSuffix() string {
```

Comment:
simplified the resolveSuffix func, but I think the behavior is also wrong. it should pick it up from the ns.Environment, not always form PublicCloud
this would change the behavior, but not in a breaking way for the consumer.
same PR or different PR?